ga/latest/kernel/helpers/build/features.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# (C) Copyright IBM Corporation 2023.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+if [ "$VERBOSE" != "true" ]; then
+  exec &>/dev/null
+fi
+
+set -Eeox pipefail 
+
+# Resolve liberty server symlinks and creation for server name changes
+/opt/ibm/helpers/build/configure-liberty.sh
+if [ $? -ne 0 ]; then
+  exit
+fi
+
+##Define variables for XML snippets source and target paths
+SNIPPETS_SOURCE=/opt/ibm/helpers/build/configuration_snippets
+SNIPPETS_TARGET=/config/configDropins/overrides
+SNIPPETS_TARGET_DEFAULTS=/config/configDropins/defaults
+mkdir -p ${SNIPPETS_TARGET}
+mkdir -p ${SNIPPETS_TARGET_DEFAULTS}
+
+# MicroProfile Health
+if [ "$MP_HEALTH_CHECK" == "true" ]; then
```

Comment:
Let's use the features.sh as an opportunity to make the users move away from [deprecated flags](https://github.com/WASdev/ci.docker#deprecated-enterprise-functionality) MP_HEALTH_CHECK, MP_MONITORING, OIDC and OIDC_CONFIG. So we don't need to add the features for those. `configure.sh` should still support them as before.

Reply:
Updated